settings.gradle

```diff
@@ -10,7 +10,7 @@ rootProject.name = 'adventure-platform-parent'
   findProject(":platform:$it")?.name = "adventure-platform-$it"
 }
 
-['bungeecord', 'spongeapi'].each {
+['bungeecord', 'spongeapi', 'legacy-text'].each {
```

Comment:
Could you rename the module to `legacy-text3`  (and directory to `text-serializer-legacy-text3`)?

Reply:
Done.